osu.Game.Rulesets.Catch/UI/CatcherArea.cs

```diff
@@ -102,6 +108,19 @@ protected override void UpdateAfterChildren()
             base.UpdateAfterChildren();
 
             comboDisplay.X = Catcher.X;
+
+            if (!lastHyperDashState && Catcher.HyperDashing && Time.Elapsed > 0)
+                CatcherTrails.DisplayEndGlow(Catcher.CurrentState, Catcher.X, Catcher.BodyScale);
```

Comment:
It looks a little funny that three/four public members of `Catcher` are accessed individually here and then passed down to the trails - although I do understand passing `Catcher` directly would result in bigger coupling...